packages/vscode-extension/src/migration/migrationTool/ts/replaceFunction.ts

```diff
@@ -0,0 +1,145 @@
+import {
```

Comment:
Curious on the file location - why it's under `ts` folder?

Reply:
The code logic of replacing JS/TS functions are shared, so just place it under `ts`.